File: Lessons/C/armstrong.c
#include <stdio.h>
#include <math.h>
int main()
{
    int sum=0, i, n, p;
    printf("Enter the Number: ") && scanf("%d", &n);

    i=n;
    while (i != 0){
    p = i%10;
    i = i/10;
    sum = sum + (p*p*p);
    }

    if (sum == n)
    {
        printf("%d is Armstrong number", n);
    }   
    else
    {
        printf("%d is not Armstrong number", n);
    }
    return 0;
}

File: Lessons/Java/armstrong.java
import java.util.Scanner;
class Arm{
    public static boolean isArmstrong(int x){
    int sum=0;
    int ognum=x;
    while(x!=0){
        int digit = x%10;
        sum += Math.pow(digit,3);
        x /= 10;
    }
    return sum == ognum;
}
}
public class armstrong {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Hello!Enter a number:");
        int x = scanner.nextInt();
        scanner.close();
        Arm res=new Arm();
        boolean result=res.isArmstrong(x);
        System.out.println(result);
    }
}
